source/includes/_tipser-tools.md

```diff
@@ -1,3 +1,29 @@
 # Tipser Tools
 
-//tools desc
\ No newline at end of file
+Tipser enables commerce on almost any site. An easy way to enable shopping on your site is to install the Tipser script and use the Tipser Tools to insert `Get The Look` widget directly into your content. The installation of the script on your site typically takes less than 5 minutes, and can be done by even non-technical people. 
+
+To get Tipser Tools visit [www.tipser.com/pos/tools](https://www.tipser.com/pos/tools) and follow the instructions there.
+
+[![](tipser-tools.png)](/images/tipser-tools.png)
+
+Once you have the Tipser Tools on your bookmark bar, simply clik it to open on top of any other site. 
+
+***
+
+## Get The Look
+The `Get the look` (GTL) enables the user to add a list of products ( or a collection  in the legacy Tipser Widget) into a post, an article or a specific area on a page. The GTL is curated by clicking the `Insert Products` in the Tipser Tools (Bookmarklet).  
```

Comment:
I think we could remove this part: `( or a collection  in the legacy Tipser Widget)`, wdyt @zdanowiczkonrad ?